x-pack/plugins/maps/common/descriptor_types/data_request_descriptor_types.ts

```diff
@@ -19,17 +19,19 @@ export type Timeslice = {
 };
 
 // Global map state passed to every layer.
-export type MapFilters = {
+export type DataFilters = {
   buffer?: MapExtent; // extent with additional buffer
   extent?: MapExtent; // map viewport
   filters: Filter[];
-  query?: MapQuery;
+  query?: Query;
   refreshTimerLastTriggeredAt?: string;
   searchSessionId?: string;
   timeFilters: TimeRange;
   timeslice?: Timeslice;
   zoom: number;
   isReadOnly: boolean;
+  searchSessionMapBuffer?: MapExtent;
+  forceRefreshTriggeredFromGlobalQueryTime?: boolean;
```

Comment:
forceRefreshTriggeredFromGlobalQueryTime is very specific but also leaves out the case where its triggered by the query bar. How about just forceRefresh?

Reply:
discussed offline, changing to `isForceRefresh`